controller.py

```diff
@@ -55,6 +56,8 @@ def run(self):
             k = readkey()
 
             if k == "q":
+                with open(self.path/"shortlist.pickle", "wb") as f:
+                    pickle.dump(self.shortlist, f)
```

Comment:
This should be a function in `model`.

Reply:
And the filename `shortlist.pickle` should be a variable in model module and used both here and in `load_shortlist`